samples/packages/pulumi-typescript/tests/index.test.ts

```diff
@@ -0,0 +1,27 @@
+import * as pulumi from'@pulumi/pulumi';
+import {describe, expect, beforeAll, beforeEach, it} from '@jest/globals';
+import * as aws from '@pulumi/aws';
+import * as helpers from './helpers';
+
+describe("OpenFABR CDF sample Pulumi Package", () => {
+    // Define the infra variable as a type whose shape matches that of the
+    // to-be-defined resources module.
+    
+    let mybucket: aws.s3.Bucket;
+
+    beforeAll(() => {
+
+    });
+
+    beforeEach(async () => {
+        mybucket = new aws.s3.Bucket("mybucket", { bucket: "my-test-bucket" });
+    });
+
+    describe("Jest setup works", () => {
+        it("test Bucket name is `my-test-bucket`", async () => {
+            // Intentionally a test independent of CDF to make sure the basic testing Jest setup works
+            const bucketName = await helpers.promiseOf(mybucket.bucket);
+            expect(bucketName).toBe("my-test-bucket");
+        });
+    });
+});
```

Comment:
Double check if this is relevant, not just something generated by pulumi but no use in this project.

Reply:
Strictly not needed but there to help troubleshooting as per the comment note above it.

Reply:
Sure, if we keep it then better have something slightly more CDF related in here instead of Pulumi's stock test. One idea: there's no point in creating a bucket here and then testing if the bucket is there; rather it could just load the whole config and run the package implementation against it, like an integration test which I don't think we have one included in this release?